packages/angular_devkit/build_angular/src/builders/browser-esbuild/tests/options/entry-points_spec.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { promises as fs } from 'fs';
+import { tmpdir } from 'os';
+import * as path from 'path';
+import { buildEsbuildBrowserInternal } from '../../index';
+import { BASE_OPTIONS, BROWSER_BUILDER_INFO, describeBuilder } from '../setup';
+
+describeBuilder(buildEsbuildBrowserInternal, BROWSER_BUILDER_INFO, (harness) => {
+  let tempDir!: string;
+
+  beforeEach(async () => {
+    tempDir = await fs.mkdtemp(path.join(tmpdir(), 'angular-cli-e2e-browser-esbuild-main-spec-'));
+  });
+
+  afterEach(async () => {
+    await fs.rm(tempDir, { recursive: true });
+  });
+
+  describe('Option: "entryPoints"', () => {
+    it('provides multiple entry points', async () => {
+      await harness.writeFiles({
```

Comment:
Can you format a little bit?

Reply:
Updated to be a little closer to how this looks in other files, hopefully that's what you meant here?